pom.xml

```diff
@@ -297,6 +297,13 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
                 <version>3.1.2</version>
+				<dependencies>
```

Comment:
I'm surprised by this addition: what is it for?

Reply:
From the CI log:

```
[INFO] --- maven-checkstyle-plugin:3.1.2:check (checkstyle) @ pac4j ---
Warning:  Old version of checkstyle detected. Consider updating to >= v8.30
Warning:  For more information see: maven.apache.org/plugins/maven-checkstyle-plugin/examples/upgrading-checkstyle.html
[INFO] You have 0 Checkstyle violations.
[INFO] 
```